tests/QRamTest.qs

```diff
@@ -114,6 +114,151 @@
         Message("Test passed.");
     }
 
```

Comment:
For this whole file, I would suggest refactoring and ensuring that you have the use submit not "nice" number of data values (i.e. not 2, 4, 8, ...) These tests are ok, but also as discussed in our standup the sorting of the data is not necessary and tests should not be on all sorted data. You can see how I have been doing this with the tests on my branch: https://github.com/qsharp-community/qram/tree/crazy4pi314/quantumdevwme/tests